mne/channels/montage.py

```diff
@@ -348,6 +347,40 @@ def get_positions(self):
         )
         return positions
 
+    def add_estimated_fiducials(self, subject, subjects_dir):
```

Comment:
The %(subject)% isn't getting expanded.
Add a verbose argument, add @verbose before this line.

Reply:
Agreed these edits should work